cmd/web3/main.go

```diff
@@ -255,6 +260,100 @@ func main() {
 							Hidden:      false},
 					},
 				},
+				{
+					Name:  "upgrade",
+					Usage: "Upgrade contract to new address",
+					Action: func(c *cli.Context) {
+						UpgradeContract(ctx, network.URL, privateKey, contractAddress, toContractAddress, amount)
+					},
+					Flags: []cli.Flag{
+						cli.StringFlag{
+							Name:        "address",
+							Destination: &contractAddress,
+							Usage:       "Proxy contract address",
+							Hidden:      false},
+						cli.StringFlag{
+							Name:        "to",
+							Destination: &toContractAddress,
+							Usage:       "Contract address to upgrade to",
+							Hidden:      false},
+						cli.IntFlag{
+							Name:        "amount",
```

Comment:
Does `deploy` need an amount flag as well?

Reply:
Maybe. That can be a separate PR though.